packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -82,6 +82,8 @@ export default declare((api, options) => {
               let context = scope.maybeGenerateMemoised(object);
               if (context) {
                 chain.object = t.assignmentExpression("=", context, object);
+              } else if (t.isSuper(context)) {
```

Comment:
```suggestion
              } else if (t.isSuper(object)) {
```
`context` is `null` in the `else` branch.

Reply:
@JLHwung Oh right, that's what I meant to do. Fixed now.